tests/pkg_littlefs2/Makefile.ci

```diff
@@ -0,0 +1,17 @@
+BOARD_INSUFFICIENT_MEMORY := 
```

Comment:
Isn't there a `\` missing at the end of this line?
```suggestion
BOARD_INSUFFICIENT_MEMORY := \
```

Reply:
Yes there is :(